media-libs/sdl2_mixer/sdl2_mixer-2.0.1.recipe

```diff
@@ -23,9 +23,10 @@ PROVIDES="
 	"
 REQUIRES="
 	haiku$secondaryArchSuffix
-	lib:libSDL2_2.0$secondaryArchSuffix
+	lib:libgl$secondaryArchSuffix
+	lib:libsdl2_2.0$secondaryArchSuffix
 	lib:libogg$secondaryArchSuffix
-	lib:libFLAC$secondaryArchSuffix
+	lib:libflac$secondaryArchSuffix
```

Comment:
I can confirm that there was an issue in Haikuporter that entries in PROVIDES did not get converted to lowercase and those in REQUIRES did (or the other way around, I don't remember), resulting in unresolved dependency to libFLAC. This should be fixed now I think, but I'd prefer if all recipes used lowercase everywhere.